common/test/Configuration.ASCforIoTLogAnalytics.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.E2ETests
```

Comment:
Is this duplicate?

Reply:
If not, what are the meanings of all those fields?

Reply:
These are configuration for the Azure security center for IoT e2e tests.
The tests query the log analytics workspace attached to the Azure security center for IoT solution in order to verify that the messages has arrived successfully.
These configurations contains the workspace id and authentication data for the workspace.
You can find more detailed explanation in ```prerequisites/readme.md```